src/utils/debounce.ts

```diff
@@ -0,0 +1,13 @@
+export function debounce<T extends unknown[]>(
+  func: (...args: T) => void,
```

Comment:
```suggestion
  func: (...args: T) => void | Promise<void>,
```

Si tu veux utiliser `async` / `await` pour `try` / `catch` à un moment